README.md

```diff
@@ -21,49 +21,67 @@ Currently we have copyFiles and copyItemsSync (both having the same parameters)
 ```
 import {copyFiles, copyItemsSync} from 'copy-files-tree'
 
-const error = copyFiles({
-    FolderPath1: FolderProps,
-    FolderPath2: { allFiles: true, destination: 'FullDestinationFolderPath'},
-    FolderPath3: { allDirectories: true, destination: 'FullDestinationFolderPath'}
-});
+const error = copyFiles(FoldersData);
 ```
 
 ## Props
```

Comment:
I think you deleted the Props from the last commit.
Btw it may be confusing that here we call it 'Props' and in the library 'Options'

Reply:
I just saw you wrote 'Options' beneath